x-pack/plugin/build.gradle

```diff
@@ -85,11 +85,6 @@ tasks.withType(RestIntegTestTask).configureEach {
 }
 
 tasks.named("yamlRestTest").configure {
-  /*
-   * We have to disable setting the number of available processors as tests in the same JVM randomize processors and will step on each
-   * other if we allow them to set the number of available processors as it's set-once in Netty.
-   */
-  systemProperty 'es.set.netty.runtime.available.processors', 'false'
```

Comment:
configuring this here probably didn't even make sense (as it's a YML test), right?

Reply:
Correct. I suspect it was just copy/pasted from elsewhere.